docs/triggers-api.md

```diff
@@ -1,11 +1,3 @@
-<!--
```

Comment:
Was this removal intentional?

Reply:
No, I executed the scripts from /hack to auto-generate these changes